shared/test/frontend_tester.py

```diff
@@ -17,14 +17,59 @@
 # need to consider whether this code wants to be shared with WebDriver tests
 # for other browsers, though.
 #
+
+gCommonDir = None
+
+# These redirects map the paths expected by the index.html files to the paths
+# in mozilla-central. In the github repo, the unit tests are run entirely within
+# karma, where the files are loaded directly. For mozilla-central we must map
+# the files to the appropriate place.
+REDIRECTIONS = {
+    "/test/vendor": "/chrome/content/shared/test/vendor",
```

Comment:
If there's some way to do avoid more hardcoded paths, since that bit us in the past, it _might_ be worth fixing or spinning off a new bug for that.  Or maybe we just don't worry about unless it becomes an issue.